lib/CatalystX/Test/MockContext.pm

```diff
@@ -35,8 +35,13 @@ sub mock_context {
     my ($req) = @_;
     my $c;
     test_psgi app => sub {
-      my $env = shift;
-      $c = $class->prepare( env => $env, response_cb => sub { } );
+        my $env = shift;
+        if (eval { $Catalyst::VERSION } >= 5.90070) {
+            Class::Load::load_class('Catalyst::Middleware::Stash');
+            $env->{Catalyst::Middleware::Stash::PSGI_KEY()} //=
```

Comment:
Please change this to use the standard Plack middleware API instead of _create_stash. This will make it more forward-compatible.

Reply:
Done.